src/Service/Configurations/RuntimeConfigProvider.cs

```diff
@@ -58,6 +58,11 @@ public virtual string RestPath
             get { return RuntimeConfiguration is not null ? RuntimeConfiguration.RestGlobalSettings.Path : string.Empty; }
         }
 
+        public virtual string RestBaseRoute
```

Comment:
Is there a reason we wouldn't have this as `string?` - should `null` be allowed, as that's valid.

Reply:
Outdated code.